src/jobs/test-package.yml

```diff
@@ -10,17 +10,24 @@ parameters:
   node_version:
     description: version of node to run tests against
     type: string
-    default: latest
+    default: '15.6.0'
   upload-coverage:
     description: if true, the job will submit code coverage report to codecov.io
     default: false
     type: boolean
+  os:
+    description: operating system you want to test against
+    type: enum
+    enum: ['linux','win/default']
+    default: 'linux'
 
 executor:
-  name: default
-  tag: << parameters.node_version >>
+  name: << parameters.os >>
 
 steps:
+  - run:
+      name: Set node version
+      command: nvm install << parameters.node_version >>
```

Comment:
when `latest` is passed in you could run `nvm install --latest-npm` 

it'd also be nice to log out the version of node that's being using just for so we're sure. For example

```
command: |
  if [ <<parameters.node_version>> = "latest" ]; then
      nvm install --latest-npm
  else
      nvm install <<parameters.node_version>>
  fi
  nvm ls
```